integrations/azure_devops.rst

```diff
@@ -0,0 +1,142 @@
+.. _azure_devops:
+
+
+|azure_logo| Azure DevOps
+=========================
+
+Thanks to the `JFrog Artifactory Extension for Azure DevOps and TFS`_ it is possible to support
+Conan tasks and integrate it with the CI development platform provided by `Microsoft’s Azure DevOps`_
+and the `Artifactory binary repository manager`_.
+
+The support for Conan now in the JFrog Artifactory Extension helps you perform the following
+tasks in Azure DevOps or TFS:
+
+ * Run Conan commands
+ * Resolve Conan dependencies from remote Artifactory servers
+ * Push Conan packages to Artifactory
+ * Publish BuildInfo metadata
+ * Import a Conan configuration
+
+In this section we will show you how to add Conan tasks to your pipelines using the Artifactory
```

Comment:
using the "Artifactory/Conan TFS Extension" (not an extension of Artifactory, but an extension to TFS)